lib/src/structs.dart

```diff
@@ -3070,106 +3364,408 @@ class ACTCTX extends Struct {
 /// If your server or container allows linking to its embedded objects, you need
 /// to register a CLSID for each supported class of objects.
 ///
 /// {@category Struct}
-class CLSID extends Struct {}
+class CLSID extends Opaque {}
 
 /// The STATSTG structure contains statistical data about an open storage,
 /// stream, or byte-array object. This structure is used in the IEnumSTATSTG,
 /// ILockBytes, IStorage, and IStream interfaces.
 ///
 /// {@category Struct}
-class STATSTG extends Struct {}
+class STATSTG extends Opaque {}
 
 /// Used to specify values that are used by SetSimulatedProfileInfo to override
 /// current internet connection profile values in an RDP Child Session to
 /// support the simulation of specific metered internet connection conditions.
 ///
 /// {@category Struct}
-class NLM_SIMULATED_PROFILE_INFO extends Struct {}
+class NLM_SIMULATED_PROFILE_INFO extends Opaque {}
 
-// typedef struct _WIN32_FIND_DATAW {
-//   DWORD    dwFileAttributes;
-//   FILETIME ftCreationTime;
-//   FILETIME ftLastAccessTime;
-//   FILETIME ftLastWriteTime;
-//   DWORD    nFileSizeHigh;
-//   DWORD    nFileSizeLow;
-//   DWORD    dwReserved0;
-//   DWORD    dwReserved1;
-//   WCHAR    cFileName[MAX_PATH];
-//   WCHAR    cAlternateFileName[14];
-//   DWORD    dwFileType;
-//   DWORD    dwCreatorType;
-//   WORD     wFinderFlags;
-// } WIN32_FIND_DATAW, *PWIN32_FIND_DATAW, *LPWIN32_FIND_DATAW;
+// typedef struct _NOTIFYICONDATAW {
+//   DWORD cbSize;
+//   HWND hWnd;
+//   UINT uID;
+//   UINT uFlags;
+//   UINT uCallbackMessage;
+//   HICON hIcon;
+//   WCHAR  szTip[128];
+//   DWORD dwState;
+//   DWORD dwStateMask;
+//   WCHAR  szInfo[256];
+//   union {
+//   UINT  uTimeout;
+//   UINT  uVersion;  // used with NIM_SETVERSION, values 0, 3 and 4
+//   } DUMMYUNIONNAME;
+//   WCHAR  szInfoTitle[64];
+//   DWORD dwInfoFlags;
+//   #endif
+//   GUID guidItem;
+//   HICON hBalloonIcon;
+//   #endif
+// } NOTIFYICONDATAW, *PNOTIFYICONDATAW;
 
-/// Contains information about the file that is found by the FindFirstFile,
-/// FindFirstFileEx, or FindNextFile function.
+// TODO: This struct does not yet support uVersion
+// FFI support for packed structs https://github.com/dart-lang/sdk/issues/38158
+
+/// The NOTIFYICONDATA contains information that the system needs to display
+/// notifications in the notification area. Used by Shell_NotifyIcon.
 ///
 /// {@category Struct}
-class WIN32_FIND_DATA extends Struct {
+class NOTIFYICONDATA extends Struct {
   @Uint32()
-  external int dwFileAttributes;
-  external FILETIME ftCreationTime;
+  external int cbSize;
+
+  @IntPtr()
+  external int hWnd;
 
-  external FILETIME ftLastAccessTime;
-  external FILETIME ftLastWriteTime;
   @Uint32()
-  external int nFileSizeHigh;
+  external int uID;
+
   @Uint32()
-  external int nFileSizeLow;
+  external int uFlags;
+
   @Uint32()
-  external int dwReserved0;
+  external int uCallbackMessage;
+
+  @IntPtr()
+  external int hIcon;
+
+  // WCHAR szTip[128]
```

Comment:
Sorry about all this crud. Inline arrays are coming real soon now (TM): https://github.com/dart-lang/sdk/issues/35763